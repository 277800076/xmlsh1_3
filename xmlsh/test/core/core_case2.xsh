# core_case2.xsh
# Test of case
i="delete"
for i in update insert delete ; do 
	echo $i 
	case $i in
	   update|insert) echo update/insert ;;
	   delete) echo delete ;;
	esac
done

