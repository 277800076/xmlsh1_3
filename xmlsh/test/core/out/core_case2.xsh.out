update
update/insert
insert
update/insert
delete
delete
